src/extension/constants.ts

```diff
@@ -11,3 +11,5 @@ export const DEFAULT_ENV = {
 export const ENV_STORE = new Map<string, string>(
   Object.entries(DEFAULT_ENV)
 )
+
+export const BOOTFILE = '.runme'
```

Comment:
Let's use a more specific file name here or have a `.runme` directory with a 
"bootfile" in it. We want to avoid overloading the `.runme` term with specific functionality. Likely iffy to change later if repos have been cloned with `.runme` files in it.

Reply:
This file will be removed after booting of VS Code has completed, so we can change its name at all times. I'ld suggest to keep as is and move it into a `.runme` directory once we decide to have something like it.